go.mod

```diff
@@ -18,3 +19,5 @@ require (
 	golang.org/x/sys v0.30.0
 	golang.org/x/time v0.10.0
 )
+
+require github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58 // indirect
```

Comment:
this dependency is not tagged nor released in 4-7 years... pinged the author on whether they could tag one